cpp/src/community/egonet.cu

```diff
@@ -63,56 +68,106 @@ extract(
 {
   auto v           = csr_view.get_number_of_vertices();
   auto e           = csr_view.get_number_of_edges();
-  auto stream      = handle.get_stream();
+  auto user_stream = handle.get_stream_view();
   float avg_degree = e / v;
   rmm::device_vector<size_t> neighbors_offsets(n_subgraphs + 1);
   rmm::device_vector<vertex_t> neighbors;
 
-  // It is the right thing to accept device memory for source_vertex
-  // FIXME consider adding a device API to BFS (ie. accept source on the device)
   std::vector<vertex_t> h_source_vertex(n_subgraphs);
-  raft::update_host(&h_source_vertex[0], source_vertex, n_subgraphs, stream);
+  std::vector<size_t> neighbors_offsets_h(n_subgraphs + 1);
```

Comment:
Better not mix h_variable_name (h_source_vertex) & variabe_name_h (neighbor_offsets_h) at least within a same function body.

Reply:
f81a2b4